downloads/index.md

```diff
@@ -1,6 +1,53 @@
 ---
 title: "Downloads"
 mirror: "https://github.com/wxWidgets/wxWidgets/releases/download"
+source_archives:
+  - description: "Windows ZIP"
+    prefix: "wxWidgets-"
+    postfix: ".zip"
+  - description: "Windows 7Z"
+    prefix: "wxWidgets-"
+    postfix: ".7z"
+  - description: "Windows Installer"
+    prefix: "wxMSW-"
+    postfix: "-Setup.exe"
+  - description: "Source for Linux, OS X, etc"
+    prefix: "wxWidgets-"
+    postfix: ".tar.bz2"
+doc_downloads:
+  - description: "Manual (HTML) ZIP"
+    postfix: "-docs-html.zip"
+  - description: "Manual (HTML) BZIP"
+    postfix: "-docs-html.tar.bz2"
+  - description: "Manual (CHM)"
+    postfix: "-docs-chm.zip"
+compilers:
+  - description: "Visual Studio 2008"
```

Comment:
Perhaps it'd better to use reverse chronological order here so that the most recent compiler is at the top?